diff.py

```diff
@@ -5,12 +5,14 @@
 import ast
 import argparse
 import subprocess
+import collections
 import difflib
 import string
 import itertools
 import threading
 import queue
 import time
+import cxxfilt
```

Comment:
as with `import Levenshtein` it might be nice to make this dependency optional to spare people the pain of installing new packages, and improving startup time a little